src/lectures/models/cmip.ipynb

```diff
@@ -0,0 +1,3361 @@
+{
```

Comment:
<p>Is this pip install necessary? Could we add it to the environment instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/earth-env-data-science/earth-env-data-science-book/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
No that was just a quick placeholder. I have a [PR](https://github.com/jbusecke/xMIP/pull/275) for this, and once it is merged we just need to make sure to update the pangeo docker image?